showcase/.eslintrc.json

```diff
@@ -0,0 +1,38 @@
+{
+  "root": true,
```

Comment:
This file could be simplified since the rules are already defined in the eslint-config:

```json
{
  "root": true,
  "extends": "@nationalbankbelgium",
  "ignorePatterns": ["src/assets/**/*"],
  "overrides": [
    {
      "files": ["*.ts"],
      "parserOptions": {
        "project": ["tsconfig.json", "e2e/tsconfig.e2e.json"],
        "createDefaultProgram": true
      },
      "rules": {
        "@typescript-eslint/no-unused-vars": ["error", { "argsIgnorePattern": "^_" }],
        "@angular-eslint/component-selector": [
          "error",
          {
            "prefix": ["app", "demo", "showcase", "stark"],
            "style": "kebab-case",
            "type": "element"
          }
        ],
        "@angular-eslint/directive-selector": [
          "error",
          {
            "prefix": ["app", "demo", "showcase"],
            "style": "camelCase",
            "type": "attribute"
          }
        ]
      }
    }
  ]
}
```